README.md

```diff
@@ -24,20 +31,36 @@ Update course advanced settings by adding `h5pxblock`
 
 ### Upload H5P Content
 
-Xblock should be available in Advanced component list of course unit now. Add xblock in unit and click "Edit" button to  
-upload H5P content and configure it.
+Xblock should be available in Advanced component list of course unit now. Add xblock in unit and click "Edit" button to upload H5P content and configure it.
 
 ![Upload and configure H5P content in Studio](https://github.com/edly-io/h5pxblock/blob/master/docs/images/upload_content.png?raw=true)
 
 ### Publish Content
 
-Use "Preview" button to preview it or publish your content and use "View Live Version" button to see how it appears  
-on LMS
+Use "Preview" button to preview it or publish your content and use "View Live Version" button to see how it appears on LMS
 
 ![Preview H5P content in LMS](https://github.com/edly-io/h5pxblock/blob/master/docs/images/preview_content.png?raw=true)
 
+### Configuring S3 as a Storage Backend
+
+When utilizing AWS S3 to store content, it is essential to configure your settings appropriately. Specifically, you must define the `H5PXBLOCK_STORAGE` setting in your common configuration file, as S3 integration does not support private buckets.
+
+Here is the required configuration:
+
+```python
+H5PXBLOCK_STORAGE = {
+    "storage_class": "storages.backends.s3boto3.S3Boto3Storage",
+    "settings": {
+        "bucket_name": "my-s3-public-bucket",
+        "querystring_auth": False,
+        "custom_domain": "my-s3-public-bucket.s3.eu-central-1.amazonaws.com"
```

Comment:
I don't think `custom domain` is a required attribute. We can skip it to here.

Reply:
Sure.